src/porepy/fracs/fracture_network_3d.py

```diff
@@ -1182,7 +1182,9 @@ def _remove_edge_intersections(
             # somewhat to obtain a more robust algorithm. Not sure about how to do
             # this consistent.
             p_new, edges_new, tags = pp.intersections.split_intersecting_segments_2d(
-                p_2d, edges_2d, tol=self.tol  # type: ignore[misc]
+                p_2d,
+                edges_2d,
+                tol=self.tol,  # type: ignore[misc]
```

Comment:
So, now we only ignore that argument, right?

Reply:
Yes, that will be the consequence, afaik.